docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/advanced-usage/creating-a-whitelabel-build/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Creating a whitelabel build"
+date: "2022-09-14"
+sidebar_position: 4000
+---
+
+```mdx-code-block
```

Comment:
For new pages, please use the <Badges/> component instead (you can git grep for Badges to see how)